docspec-python/.changelog/_unreleased.toml

```diff
@@ -0,0 +1,21 @@
+# NOTE (@nrser) Obviously these files have some format/structure to them but
+#   I'm not familiar with it and I'm not sure where to find it documented;
+#   please forgive any mistakes.
+
+# Guessing this would not be set in an "unreleased" entry, but anyways today's
+# date is thrown in there
+# 
+# release-date = "2023-02-28"
+
+[[entries]]
+# Guessing this is just a uuid, I generated one with `uuidgen`
+id = "58c1a773-c483-4014-8238-80d9f65f7b57"
+# I guess this is a 'fix'? Python version for `docspec-python` is not restricted
+# above, so I assume it is supposed to work for Python 3.10+ with `match`
+type = "fix"
+description = """\
+  Swap in `blib2to3` parser (bundled with `black` package) for the stdlib \
+  `lib2to3` module in order to support `match` statement (PEP 634 – Structural \
+  Pattern Matching). \
+"""
+author = "@nrser"
```

Comment:
Sorry that it's not documented here :) The format is about right; release date doesn't go into unreleased and the id needs to be unique as you guessed.

Usually this entry would be created with

```
slap changelog add -t fix -d 'Swap in `blib2to3` parser ...'
```

(It should autodetect your GitHub username from your configured Git email)